java/src/jmri/jmrit/display/switchboardEditor/BeanSwitch.java

```diff
@@ -43,7 +43,7 @@
     private int square = 75; // outside dimension of graphic, normally < 2*radius
     private int radius = 50; // max distance in px from center of switch canvas, unit used for relative scaling
     private double popScale = 1.0d;
-    private Boolean showUserName = true;
+    private int showUserName = 1;
```

Comment:
Might be useful to add a comment here about what 1 and 2 mean (are those the only valid values?)